### ISSUE TITLE
Fix remaining code style issues

### ISSUE THREAD

**Issue by pixeldesu** [surfbirdapp/surfbird#25]
Run `npm run lint` or look at the following list to see specific coding style problems:

```
app/_js/main.js:144:5: Do not use 'new' for side effects.
app/_js/main.js:144:9: 'Notification' is not defined.

main.js:30:7: 'mainWindow' is not defined.

src/storage/keys.js:1:42: Use path.join() or path.resolve() instead of + to create paths.

src/windows/main.js:20:22: Use path.join() or path.resolve() instead of + to create paths.
src/windows/main.js:40:7: 'stream' is defined but never used.
```

**Comment by pixeldesu** [comment]
Just fixed a bunch of the `...is not defined` errors with adding globals in f27100ac7d6be67a3b2d44741bef1cff760fcabf

**Comment by pixeldesu** [comment]
Fixed some of the unused reference errors in 1ef1cc1

**Comment by pixeldesu** [comment]
Removed some unused gulp dependencies in 88e9211

**Comment by pixeldesu** [comment]
Added proper error handlers in 7c1bc4c537e25398fe673f068e54015010bcb12a

**Comment by pixeldesu** [comment]
Fixed the comparison operators in 003f024a354e83d371ea08aa98ed94ed3cc231f4

**Comment by pixeldesu** [comment]
Fixed unitialized constants and variables in 65032ef

**Comment by pixeldesu** [comment]
Fixed some more issues in 72e02a9dfa33027a74fd152054b24f021b7325a5, heading for the final stretch

**Comment by pixeldesu** [comment]
Everything back on green!
